cohort_4/playbook/workshops/culture_tech_cadets_portfolio/README.md

```diff
@@ -0,0 +1,490 @@
+# Building Your First Website
+
+In this tutorial, you will be learning how to build your first website!
+
+There are two parts of this tutorial:
+
+In part 1, we'll be using the website [Dash](dash.generalassemb.ly) to learn the
```

Comment:
You need to include the protocol for the link to work, else GitHub thinks you're linking to a file. This link is currently broken.